source/fundamentals/connection/socks5.txt

```diff
@@ -0,0 +1,113 @@
+.. _kotlin-connect-socks:
+
+==========================================
+Connect to MongoDB by Using a SOCKS5 Proxy
+==========================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, security, connection string
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to connect
+to MongoDB by using a **SOCKS5 proxy**. SOCKS5 is a standardized
+protocol for communicating with network services through a proxy server.
+
+.. tip::
+
+   To learn more about the SOCKS5 protocol, see the Wikipedia entry on
+   :wikipedia:`SOCKS <w/index.php?title=SOCKS&oldid=1160087146>`.
+
+.. _kotlin-socks-proxy-settings:
+
+SOCKS5 Proxy Settings
+---------------------
+
+The proxy settings specify the SOCKS5 proxy server address and your
+authentication credentials. You can specify your settings in an instance of
+``MongoClientSettings`` or in your connection string.
+
+The following table describes the SOCKS5 client options:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 20 65
+
+   * - Name
+     - Accepted Values
+     - Description
+
+   * - **proxyHost**
+     - String
+     - Specifies the SOCKS5 proxy IPv4 address, IPv6 address, or hostname.
+       You must provide this value to connect to a SOCKS5 proxy.
+
+   * - **proxyPort**
+     - non-negative Integer
+     - Specifies the TCP port number of the SOCKS5 proxy server. This option
+       defaults to ``1080`` when you set ``proxyHost``.
+
+   * - **proxyUsername**
+     - String
+     - Specifies the username for authentication to the SOCKS5 proxy server.
+       The driver ignores ``null`` and empty string values for this setting.
+       The driver requires that you pass values for both ``proxyUsername``
+       and ``proxyPassword`` or that you omit both values.
+
+   * - **proxyPassword**
+     - String
+     - Specifies the password for authentication to the SOCKS5 proxy server.
+       The driver ignores ``null`` and empty string values for this setting.
+       The driver requires that you pass values for both ``proxyUsername``
+       and ``proxyPassword`` or that you omit both values.
+
+
+Examples
+--------
+
+The following examples show how to instantiate a ``MongoClient`` that connects
+to MongoDB by using a SOCKS5 proxy. The proxy settings can be specified in a
+``MongoClientSettings`` instance or a connection string. These examples use
+the placeholder values described in the :ref:`kotlin-socks-proxy-settings` section.
+Replace the placeholders with your proxy specifications and credentials.
+
+Specify Proxy Settings in the MongoClientSettings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code example shows how to specify your SOCKS5 proxy settings by
+using the ``applyToSocketSettings()`` builder method when creating a
+``MongoClientSettings``:
```

Comment:
S/nit: I think "creating a mongoclientsettings" sounds like incorrect grammar or an incomplete sentence, even though it's technically fine since that's just the name of the instance being created

```suggestion
using the ``applyToSocketSettings()`` builder method when creating a
``MongoClientSettings`` instance:
```